trl/trainer/grpo_config.py

```diff
@@ -241,14 +241,40 @@ class GRPOConfig(TrainingArguments):
             </Deprecated>
 
         vllm_importance_sampling_correction (`bool`, *optional*, defaults to `True`):
-            Whether to apply Truncated Importance Sampling (TIS) between vLLM completion logprobs and recomputed
-            logprobs. [Your Efficient RL Framework Secretly Brings You Off-Policy RL
-            Training](https://fengyao.notion.site/off-policy-rl) highlights that using a separate generation framework
-            (such as vLLM) can introduce off-policy effects due to subtle implementation differences between generation
-            and training backends. TIS is proposed as a remedy for this issue.
+            Whether to apply Importance Sampling (IS) to correct for the mismatch between vLLM
+            completion logprobs and recomputed training logprobs. If set to `False`, no IS is applied
+            regardless of `vllm_importance_sampling_mode`. When `True`, the selected mode determines
+            how the IS ratios are computed and constrained.
+
+        vllm_importance_sampling_mode (`str`, *optional*, defaults to `"sequence_mask"`):
+            Specifies how Importance Sampling is performed when `vllm_importance_sampling_correction=True`.
+
+            The mode is defined along two orthogonal dimensions:
+            * Constraint: how to handle importance ratios above `vllm_importance_sampling_cap` (C):
+                 - truncation: clip ratios from above, ρ ← min(ρ, C), as in
+                [Your Efficient RL Framework Secretly Brings You Off-Policy RL Training](https://fengyao.notion.site/off-policy-rl)
+                - masking: set ratios above C to zero, so those contributions do not affect the gradient, as in
+                [When Speed Kills Stability: Demystifying RL Collapse from the Training-Inference Mismatch](https://yingru.notion.site/When-Speed-Kills-Stability-Demystifying-RL-Collapse-from-the-Training-Inference-Mismatch-271211a558b7808d8b12d403fd15edda)
+
+            * Granularity: the level at which ratios are computed:
+                - token: per-token ratios ρ_t
+                - sequence: a single ratio ρ_seq per sequence, applied to all tokens
+
+            Supported options are:
+                - `"token_truncate"`:
+                    Token-level truncated IS (default). Per-token ratios are clipped from above at C.
+                - `"token_mask"`:
+                    Token-level masked IS. Per-token ratios above C are set to zero.
+                - `"sequence_truncate"`:
+                    Sequence-level truncated IS. A single sequence ratio is clipped from above at C and
+                    applied to all tokens in the sequence.
+                - `"sequence_mask"`:
+                    Sequence-level masked IS. Sequences with ratios above C are masked out.
+
```

Comment:
This is a bit long for having it in the doctoring in my opinion. I'd suggest this instead:

```suggestion
        vllm_importance_sampling_mode (`str`, *optional*, defaults to `"sequence_mask"`):
            Specifies how Importance Sampling is performed when `vllm_importance_sampling_correction=True`. Possible
            values are:

                - `"token_truncate"`: Token-level truncated IS (default). Per-token ratios are clipped from above at C.
                - `"token_mask"`: Token-level masked IS. Per-token ratios above C are set to zero.
                - `"sequence_truncate"`: Sequence-level truncated IS. A single sequence ratio is clipped from above at
                  C and applied to all tokens in the sequence.
                - `"sequence_mask"`: Sequence-level masked IS. Sequences with ratios above C are masked out.
```

and create a dedicated subsection, like "Dealing with the Training-Inference Mismatch" in grpo_trainer.md, where you'd copy the above description